scholarx/archive/index.html

```diff
@@ -88,33 +88,25 @@ <h1 class="mb-0 text-onelive">Take a look at our past</h1>
                 <p class="display-3 text-onelive font-weight-bolder">ScholarX</p>
             </div>
         </div>
-        <div class="row">
-            <div class="col-md-6">
-                <img class="card-img card-template"
-                     src="images/card-image.jpg">
-                <div class="card">
-                    <div class="card-body">
-                        <h5 class="card-title">ScholarX 2020</h5>
-                        <p class="card-text">We expanded the programme to 18 mentors and over 50 mentees.
-                            Our mentors were once again from Fortune500 companies and top Universities.</p>
-                        <a href="2020"
-                           class="card-link">View More</a>
-                    </div>
+        <div class="card-deck">
+            <a href="2020" class="card col-md-6 shadow project-card" style="padding: 0px;">
+                <img src="images/card-image.jpg" class="card-img card-template" style="border-radius: 25px 25px 0px 0px;" alt="2020">
```

Comment:
Can we move these inline CSS to an external CSS file??
I think you can use the global custom CSS file for stuff like this.